### PR TITLE
checks if url changes i.e. card opens, to fix 'child' & 'parent' butt…

### PR DESCRIPTION
**Pull request by BennetLeff**
…ons not showing

### CODE REVIEW

File: src/js/plugins/inheritance.js [hunk 1]
```diff
@@ -1632,7 +1632,34 @@
...
+              // page has changed, set new page as 'current'
+              currentPage = window.location.href;
+
+              // update only if a card is open (which we can tell by checking if there is a /c/ in the url)
+              if (currentPage.includes("/c/"))
+              {
+                self.updateCardView();
+              }
+          }
+      }, 100);
```
Comment (nicholaspsmith) [inline src/js/plugins/inheritance.js:1661]: See if you can find a way to detect url change without having to poll every 100ms

Comment (SeanMcTex) [reply src/js/plugins/inheritance.js:1661]: Surprisingly, it looks like reliable ways to detect the URL change are in short supply. https://stackoverflow.com/questions/6390341/how-to-detect-url-change-in-javascript

File: src/js/plugins/inheritance.js [hunk 2]
```diff
@@ -1632,7 +1632,34 @@
       }
     },
 
+    /*
+     * The init function is the first code the chrome plugin will run before 
+     * doing anything.
+     */
     init: function () {
+      /*
+       * The following code checks if a any chagne is made to the url indicating
```
Comment (SeanMcTex) [inline src/js/plugins/inheritance.js:1641]: "…if any change…" #nitpick

**Comment by sdbrown2** [comment]
quick work, thanks bennet!

**Pull request closed without merging**